index.html

```diff
@@ -59,19 +64,124 @@ <h3>
-                <li><a href="#"><img src="images/github.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="images/linkedin2.png" alt="linkedin icon"></a></li>
-                <li><a href="#"><img src="images/two.png" alt=""></a></li>
-                <li><a href="#"><img src="images/twitter.png" alt="twitter icon"></a></li>
-                <li><a href="#"><img src="images/medium.png" alt="medium logo"></a></li>
+        <section id="portfolio">
+            <p class="portfolio-heading">My Recent Work</p>
+            <hr>
+            <div class="portfolio-img">
+                <img src="images/placeholder.png" alt="">
+            </div>
+            <h2>
+                Multi-Post Stories
+            </h2>
+            <p class="portfolio-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                required. has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="skill-list">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
             </ul>
+            <a href="#" class="btn skill-list-btn">See Project</a>
+        </section>
+
+        <section class="printing">
+            <div class="">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard</p>
+                <ul class="skill-list printing-list">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+
+            <a href="#" class="btn btn_2">See Project</a>
+
+        </section>
+
+        <section class="printing">
+            <div class="">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard</p>
+                <ul class="skill-list printing-list">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+
+            <a href="#" class="btn btn_2">See Project</a>
+
         </section>
+
+
+    </div>
+
+    <section id="about-me">
+        <h1 class="container">About me</h1>
+        <p class="container">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.</p>
+        <a href="#" class="btn skill-list-btn container">Get My Resume</a>
+        <img src="images/Group.png" alt="">
```

Comment:
- [ ] Decorations should be background-images (in CSS), not `img` tags. Kindly remove this image here and implement the decoration using CSS.